charts/polaris/charts/aws-service-operator/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: aws-service-operator
+version: 0.1.0
```

Comment:
Fix this chart also - this is a Polaris owned chart (does not exist as a chart in the upstream aws-service-operator repo - maybe it will in the future, so this is a stop-gap)

Reply:
What else should go in here besides a better description?